src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -1,16 +1,79 @@
 (ns status-im2.contexts.wallet.create-account.view
-  (:require [react-native.core :as rn]
-            [quo2.core :as quo]
-            [re-frame.core :as rf]))
+  (:require
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.core :as quo]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.wallet.create-account.style :as style]
+    [utils.re-frame :as rf]))
 
-(defn view
+(defn- view-internal
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
+  (let [top                  (safe-area/get-top)
+        bottom               (safe-area/get-bottom)
+        color                (reagent/atom :blue)
+        emoji                (reagent/atom "\uD83D\uDC8E") ; diamond emoji
```

Comment:
consider:
```
(def diamond-emoji "\uD83D\uDC8E")

(defn- view-internal
  []
  (let [top                  (safe-area/get-top)
        bottom               (safe-area/get-bottom)
        color                (reagent/atom :blue)
        emoji                (reagent/atom diamond-emoji) ; 
```
imo its better to use code that speaks than comments that describe.

Reply:
same here, how about ` account-emoji                (reagent/atom diamond-emoji) ; `